koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -546,40 +353,10 @@ def handler(self, src, target_info, arch, output_template, scratch=False,
         self.logger.debug("OSBS build finished with status: %s. Build "
                           "response: %s.", response.status,
                           response.json)
-        logs = self._download_logs(build_id)
 
         self.logger.info("Response status: %r", response.is_succeeded())
 
-        files = []
-        if response.is_succeeded() and response.get_tar_metadata_filename():
-            sourceimage = response.get_tar_metadata_filename()
-            targetimage = self._getOutputImageTemplate(arch)
-
-            files = self._download_files(sourceimage, targetimage)
-
-        rpmlist = []
-        if response.is_succeeded():
-            rpmlist = self._get_rpm_packages(response)
-
-        self.logger.debug("rpm list:")
-        for rpm in rpmlist:
-            self.logger.debug("%(name)s-%(version)s-%(release)s.%(arch)s.rpm" %
-                              rpm)
-
         repo_info = self.session.getRepo(target_info['build_tag'])
```

Comment:
I actually think that was out previously and somehow snuck in because I missed it during a rebase. Will fix.